File: app/src/main/java/br/com/disapps/homepet/ui/home/adapter/HotelAdapter.java
package br.com.disapps.homepet.ui.home.adapter;

import android.support.annotation.Nullable;

import com.chad.library.adapter.base.BaseQuickAdapter;

import java.util.List;

import br.com.disapps.homepet.R;
import br.com.disapps.homepet.data.model.Hotel;
import br.com.disapps.homepet.ui.custom.CustomViewHolder;

/**
 * Created by diefferson.santos on 23/08/17.
 */

public class HotelAdapter extends BaseQuickAdapter<Hotel, CustomViewHolder> {

    public HotelAdapter(@Nullable List<Hotel> data) {
        super(R.layout.hotel_item, data);
    }

    @Override
    protected void convert(CustomViewHolder helper, Hotel item) {
        helper.setImageURI(R.id.hotel_image, item.getUrlImagemCapa());
        helper.setText(R.id.hotel_name, item.getNome());
    }
}

File: presentation/src/main/java/br/com/disapps/homepet/ui/hotels/adapter/HotelAdapter.kt
package br.com.disapps.homepet.ui.hotels.adapter

import br.com.diefferson.domain.model.Hotel
import com.chad.library.adapter.base.BaseQuickAdapter
import br.com.disapps.homepet.R
import br.com.disapps.homepet.ui.custom.CustomViewHolder

/**
 * Created by diefferson.santos on 23/08/17.
 */

class HotelAdapter(data: List<Hotel>?) : BaseQuickAdapter<Hotel, CustomViewHolder>(R.layout.hotel_item, data) {

    override fun convert(helper: CustomViewHolder, item: Hotel) {
        helper.setText(R.id.hotel_name, item.name)
        helper.setImageURI(R.id.hotel_image, item.coverImage)
        helper.setRating(R.id.rating, item.rating)
        helper.setText(R.id.ratings, item.ratingsNumber.toString())
        helper.setText(R.id.comments, item.commentsNumber.toString())
        helper.setText(R.id.price, item.price.toString())
        helper.setText(R.id.distance, item.address)
    }
}
